packages/astro/src/default/components/OpenInStackblitzLink.astro

```diff
@@ -0,0 +1,93 @@
+<button
+  transition:persist
+  title="Open in StackBlitz"
+  data-id="open-in-stackblitz"
+  class="flex items-center font-size-3.5 text-tk-elements-topBar-iconButton-iconColor hover:text-tk-elements-topBar-iconButton-iconColorHover transition-theme bg-tk-elements-topBar-iconButton-backgroundColor hover:bg-tk-elements-topBar-iconButton-backgroundColorHover p-1 rounded-md"
+>
+  <svg viewBox="0 0 28 28" aria-hidden="true" height="24" width="24">
+    <path fill="currentColor" d="M12.747 16.273h-7.46L18.925 1.5l-3.671 10.227h7.46L9.075 26.5l3.671-10.227z"></path>
+  </svg>
+</button>
+
+<script>
+  import StackBlitzSDK from '@stackblitz/sdk';
+  import { tutorialStore } from '../components/webcontainer.js';
+
+  // initialize handlers on each page load as it's possible some pages disable openInStackBlitzLink
+  document.addEventListener('astro:page-load', onInit);
```

Comment:
This is required even when `transition:persist` is used due to:

- Disable `openInStackBlitzLink` on a single lesson
- Open that lesson in fresh browser tab. The element is not run.
- Navigate to a page that doesn't have `openInStackBlitzLink` disabled. Script tags are not run, handler is not attached.